gen_openj9_templates.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+
+# Remove old openj9 templates
+rm templates/openj9/*
+
+for original_template in templates/*.json; do
+  echo Creating OpenJ9 template for file "$original_template"
+  sed -e 's/OpenJDK/OpenJ9/' \
+    -e 's/sso74/sso74-openj9/' \
+    "$original_template" \
+    | tee "$(echo "$original_template" | sed -e 's_templates_templates/openj9_' -e 's/\([a-zA-Z0-9]*\)-/\1-openj9-/')" >/dev/null
```

Comment:
```suggestion
    > "$NEW_TEMPLATE"
```

Reply:
+1 Redirect the output into the `"templates/openj9/$original_template"` You can even move it later to proper name if needed. No need to do all the actions in one step. Rather to keep each step atomic.

Or alternatively pre-generate the new template name name first into `$NEW_TEMPLATE` as part of each for loop iteration, and write it there directly.